app/models/whitehall_migration/asset_import.rb

```diff
@@ -9,6 +9,12 @@ class WhitehallMigration::AssetImport < ApplicationRecord
 
   validate :associated_with_only_image_or_file_attachment
 
+  def asset_manager_id
```

Comment:
This isn't actually an asset manager id - this is the id whitehall assigns for the asset. Might be better named as whitehall_asset_id